sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/KnownLinuxVirtualMachineImage.java

```diff
@@ -41,7 +41,15 @@ public enum KnownLinuxVirtualMachineImage {
     REDHAT_RHEL_8_2("RedHat", "RHEL", "8.2"),
 
     /** Oracle Linux 8.1. */
-    ORACLE_LINUX_8_1("Oracle", "Oracle-Linux", "81");
+    ORACLE_LINUX_8_1("Oracle", "Oracle-Linux", "81"),
+
+
+    /** UbuntuServer 18.04LTS Gen2. */
+    UBUNTU_SERVER_18_04_LTS_GEN2("Canonical", "UbuntuServer", "18_04-lts-gen2"),
+    /** UbuntuServer 20.04LTS. */
+    UBUNTU_SERVER_20_04_LTS("Canonical", "0001-com-ubuntu-server-focal", "20_04-lts"),
+    /** UbuntuServer 20.04LTS Gen2. */
+    UBUNTU_SERVER_20_04_LTS_GEN2("Canonical", "0001-com-ubuntu-server-focal", "20_04-lts-gen2");
```

Comment:
nit, "0001-com-ubuntu-server-focal" seems a bit strange...

Reply:
Indeed... There are others complaining about this too:
https://github.com/Azure/azure-cli/issues/13320#issuecomment-665794215

But this is the actual default the Portal uses. And no offer with name `UbuntuServer` exists after 20.04. I think we have to accept it...